packages/patternfly-4/react-core/src/components/OptionsMenu/OptionsMenu.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import styles from '@patternfly/patternfly/components/OptionsMenu/options-menu.css';
+import { css, getModifier } from '@patternfly/react-styles';
+
+export enum OptionsMenuPosition {
+  right = 'right',
+  left = 'left'
+}
+
+export enum OptionsMenuDirection {
+  up = 'up',
+  down = 'down'
+}
+
+export interface OptionsMenuProps extends React.HTMLProps<HTMLDivElement>{
+  /** Classes applied to root element of the Options menu */
+  className?: string;
+  /** Id of the root element of the Options menu */
+  id: string;
+  /** Array of OptionsMenuItem and/or OptionMenuItemGroup nodes that will be rendered in the Options menu list */
+  menuItems: React.ReactNode[];
+  /** Either an OptionsMenuToggle or an OptionsMenuToggleWithText to use to toggle the Options menu */
+  toggle: React.ReactElement;
```

Comment:
Hold tight while we figure out what this type should be.

Reply:
@redallen @dlabaj 
This cannot be a ReactNode because it is used later in React.cloneElement (line 55). That function expects type ReactElement